src/services/encryptPassword.js

```diff
@@ -3,7 +3,7 @@ const saltRounds = 10;
 
 module.exports.encryptPassword = async function (password) {
     let pass;
-    await bcrypt.hash(password, saltRounds).then(function(hash) {
+    await bcrypt.hash(password, saltRounds).then(function (hash) {
```

Comment:
Also here, arrow function

```suggestion
    await bcrypt.hash(password, saltRounds).then(hash => {
```

There is also a function named comparePassword in this file, down below, please use also arrow function there, for some reason I can't add comment on the exact line :)